apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/RevertFinalDecisionDialog.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { fakePoolCandidates } from "@gc-digital-talent/fake-data";
+import { OverlayOrDialogDecorator } from "@gc-digital-talent/storybook-helpers";
+import {
+  PoolCandidateStatus,
+  makeFragmentData,
+} from "@gc-digital-talent/graphql";
+
+import RevertFinalDecisionDialog, {
+  RevertFinalDecisionDialog_Fragment,
+} from "./RevertFinalDecisionDialog";
+
+const fakedCandidate = fakePoolCandidates(1)[0];
+const qualifiedData = makeFragmentData(
+  { ...fakedCandidate, status: PoolCandidateStatus.QualifiedAvailable },
+  RevertFinalDecisionDialog_Fragment,
+);
+const disqualifiedData = makeFragmentData(
+  { ...fakedCandidate, status: PoolCandidateStatus.ScreenedOutAssessment },
+  RevertFinalDecisionDialog_Fragment,
+);
+
+export default {
+  component: RevertFinalDecisionDialog,
+  title: "Components/Revert final decision dialog",
```

Comment:
One last thing, we decided to no longer do titles and just let storybook auto generate :sweat_smile: 

https://github.com/GCTC-NTGC/gc-digital-talent/pull/10171#issuecomment-2077525896

Reply:
Fixed by [a1710e0](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10152/commits/a1710e051cbf116277e23d8e61d46f584100a1ca)